pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -127,77 +123,111 @@ def _base(self):
 
         return self.__base
 
+    @property
```

Comment:
I believe in anaconda codebase there is unwritten rule that @properties are readonly. Here you are doing a lazy initialization of this property, therefore it's not readonly. 

I suggest, to either create a setter function or use https://www.geeksforgeeks.org/python-functools-cached_property/ in order for the None checks to be skipped.

@M4rtinK do you know if we have a convention for this?